packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -111,5 +111,10 @@ export const messages = {
     'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.',
   force_project_create_text: 'SFDX: Create Project',
   force_project_create_open_dialog_create_label: 'Create Project',
-  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger'
+  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger',
+  force_start_apex_debug_logging:
+    'SFDX: Turn on Apex Debug Log for Replay Debugger',
+  force_apex_debug_log_status_bar_text: 'Logging ends at %s',
```

Comment:
@Sweetman -
Per my conversation on Slack with @vazexqi - 
1. Please add the file icon (https://octicons.github.com/icon/file-text). 
2. Please change this message to: "Recording detailed logs until <time>"
3. Please use the date/time format for en_US that's listed here: https://help.salesforce.com/articleView?id=admin_supported_locales.htm&type=5
3. Please add hover text: "Writing debug logs for Apex and Visualforce at the FINEST log level"